### PR TITLE
Added a text file

### PR DESCRIPTION
**Pull request by iswarya-murali-brainlabs**

### CODE REVIEW

File: iswarya.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+first text file upload
```
Comment (deepanshuBrainlabs) [inline iswarya.txt:1]: you should add more text

Comment (iswarya-murali-brainlabs) [reply iswarya.txt:1]: I have added more text

Comment (ShrutiBrainlabs) [reply iswarya.txt:1]: Ok I wil add. Thanks for pointing it out

Comment (deepanshuBrainlabs) [reply iswarya.txt:1]: this PR is no more required .

**Review by ShrutiBrainlabs — APPROVED** [review]

**Pull request closed without merging**